src/builder-post-process.c

```diff
@@ -28,7 +28,7 @@
 #include <sys/statfs.h>
 
 #include <gio/gio.h>
-#include "libglnx/libglnx.h"
+#include "libglnx.h"
```

Comment:
Why did the include path change? Not familiar with this lib

Reply:
Because this is the correct path. Meson adds strictly the correct include path here, whereas Autotools does that AND adds a catch-all include on subprojects. I'm not going to change this on Autotools now.